x-pack/solutions/security/plugins/security_solution/public/common/lib/telemetry/events/siem_migrations/types.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { RootSchema } from '@kbn/core/public';
+
+export enum SiemMigrationsEventTypes {
+  TranslatedRuleUpdate = 'Translated Rule Update',
```

Comment:
Nit: might be better if these were camelcased because they are almost treated as IDs.

Reply:
I checked and all events in `x-pack/solutions/security/plugins/security_solution/public/common/lib/telemetry/events` are done like this. Let me know if you prefer the camelcased, we can change it before feature is released.